ajax/libs/vega/package.json

```diff
@@ -13,8 +13,13 @@
     {
       "basePath": "",
       "files": [
-        "vega.js",
-        "vega.min.js"
+        "vega*.+(js|js.map)"
+      ]
+    },
+    {
+      "basePath": "build",
+      "files": [
+        "vega*.+(js|js.map)"
```

Comment:
What do you think about just using `"**/vega*"` which can grab files under different directories with the same directory/file structure?